server/server.go

```diff
@@ -1,35 +1,43 @@
 package server
 
 import (
+	"errors"
 	"fmt"
 	"net/http"
-	"time"
+	"strings"
 )
 
-// Middleware is a function that decorates an http.Handler
-type Middleware func(handler http.Handler) http.Handler
+var (
+	// SupportedAuthTypes is a whitelist of supported authentication types. Deafult: Bearer
```

Comment:
please delete now deprecated values for the below from https://github.com/tozny/utils-go/blob/master/http.go